src/app/components/merge/MergeModalContent.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Alert } from 'react-components';
+
+import MergeTable from './MergeTable';
+
+const MergeModalContent = ({
+    onSortEnd,
+    orderedContacts,
+    isChecked,
+    isDeleted,
+    onClickCheckbox,
+    onClickDetails,
+    onClickDelete,
+    onClickUndelete,
+    onClickPreview
+}) => {
+    return (
+        <>
+            <Alert>
+                {c('Description')
+                    .jt`Use Drag and Drop to rank merging priority between contacts. Uncheck the contacts you do ${(
+                    <b key="boldface">not</b>
```

Comment:
Inline jsx is invalid in ttag :/ https://ttag.js.org/docs/jsx-gettext.html#invalid

Reply:
So the suggestion from there is to have sth like
```
const boldNot = <b key="boldface">{c('Description').t`not`}</b>;
...
<Alert>{c('Description').jt`Use Drag and Drop to rank merging priority between contacts. Uncheck the contacts you do ${boldNot} want to merge`}</Alert>
```
I wonder if that will be understandable in a platform like Crowdin

Reply:
I mean we should follow `ttag`s best practice and assume it handles that